actions/actionsMatchtxt_test.go

```diff
@@ -55,7 +56,9 @@ tuning:                   not tuned
 Remember: if you wish to automatically activate the note's and solution's tuning options after a reboot, you must enable saptune.service by running:
  'saptune service enable'.
 
-`, system.GetTunedAdmProfile(), system.GetVirtStatus())
+`, "balanced", system.GetVirtStatus())
+
+//`, system.GetTunedAdmProfile(), system.GetVirtStatus())
```

Comment:
Is this something that you intentionally leave? I mean the comment of the old parameters!

Reply:
no, was a left over. Thanks for pointing this out, fixed with new committed code